Controllers/UserController.cs

```diff
@@ -0,0 +1,140 @@
+﻿using API.Models;
+using API.Models.DTO;
+using API.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.Controllers
+{
+    [Route("api/v1/users")]
+    [ApiController]
+    [Produces(ApiContentType)]
+    public class UserController : ApiControllerBase
+    {
+        private readonly UserManager<User> _userManager;
+        private readonly SignInManager<User> _signInManager;
+        private readonly RoleManager<IdentityRole> _roleManager;
+        private readonly JwtService _jwt;
+
+        public UserController(
+            ApiContext context,
+            IConfiguration config,
+            UserManager<User> userManager,
+            SignInManager<User> signInManager,
+            RoleManager<IdentityRole> roleManager) :
+            base(context)
+        {
+            _userManager = userManager;
+            _signInManager = signInManager;
+            _roleManager = roleManager;
+            _jwt = new JwtService(config);
+        }
+
+
+        [HttpPost("login")]
+        [AllowAnonymous]
+        public async Task<IActionResult> Login(AuthDTO model)
+        {
+            var user = await _userManager.FindByEmailAsync(model.Email);
```

Comment:
Request validation check is missing. We always do this check first to check headers:
```C#
if (!IsValidApiRequest())
{
  return ApiBadRequest("Invalid Headers!");
}
```